SwiftUI_DZ/SwiftUI_DZ/ContentView.swift

```diff
@@ -6,12 +6,194 @@
+        static let myNameText = "Ernest"
+        static let settingsText = "Apple ID, iCloud, контент и покупки"
+        static let smallCircleImageName = "1.circle.fill"
+        static let wifiTextName = "Wi-Fi"
+        static let imageWidthFrameCount: CGFloat = 30
+        static let imageHeightFrameCount: CGFloat = 30
+        static let cornerRadiusNumber: CGFloat = 5
+        static let circleSize: CGFloat = 70
+        static let settingsTextFontSize: CGFloat = 12
+        static let vStackSpacing: CGFloat = 20
+    }
+    
+    @State private var isOn = false
+    @State private var isWiFiSelection = 0
+    @State private var isBluetoothSelection = 0
+    @State private var isVPNSelection = 0
+    @State private var string = ""
+    
     var body: some View {
-        Text("Hello world")
+        NavigationStack {
+            VStack(spacing: Constants.vStackSpacing) {
+                Form {
+                    Section {
+                        setupAboutUser()
+                        setupAboutAppleIDView()
+                    }
+                    
+                    Section {
+                        setupCancelSettingView()
+                    }
+                    
+                    Section {
+                        setupAirplaneToggle()
+                        setupWiFiPicker()
+                        setupBluetoothPicker()
+                        setupMobileData()
+                        setupPersonalHotspot()
+                        setupVPNPicker()
+                    }
+                }
+                .navigationTitle(Constants.navigationTitleText)
+            }
+            .searchable(text: $string)
+        }
+        
+    }
+    
+    fileprivate func setupAboutAppleIDView() -> some View {
+        HStack {
+            Text(Constants.appleIDText)
+            Spacer()
+            Image(systemName: Constants.rightChevronImageName)
+        }
+    }
+    
+    fileprivate func setupAboutUser() -> some View {
+        HStack {
+            Circle()
+                .frame(width: Constants.circleSize, height: Constants.circleSize)
+                .foregroundColor(.red)
+            VStack {
+                HStack {
+                    Text(Constants.myNameText)
+                        .font(.largeTitle)
+                    Spacer()
+                }
+                Text(Constants.settingsText)
+                    .font(.system(size: Constants.settingsTextFontSize))
+            }
+            Spacer()
+            Image(systemName: Constants.rightChevronImageName)
+        }
+    }
+    
+    fileprivate func setupCancelSettingView() -> some View {
+        HStack {
+            Text(Constants.cancelSettings)
+            Spacer()
+            Image(systemName: Constants.smallCircleImageName)
+                .foregroundColor(.red)
+            Image(systemName: Constants.rightChevronImageName)
+        }
+    }
+    
+    fileprivate func setupPersonalHotspot() -> some View {
```

Comment:
```suggestion
    private var setupPersonalHotspot: some View {
```
и ниже везде исправить функции без параметров на вычисляемые свойства